client/index.jsx

```diff
@@ -83,9 +83,10 @@ class Reviews extends React.Component {
   fetchListingInfo(urlID) {
     //get review rating categories
     $.ajax({
-      url: `http://18.217.248.48:3000/api/overall_reviews/${urlID}`,
+      url: `/api/overall_reviews/${urlID}`,
       method: 'GET',
       success: (reviews) => {
+        console.log(reviews);
```

Comment:
Oops accidentally merged a console.log() 🙃. Lint is going to be so mad at me. I'll take this out next commit.

Reply:
I love how you picked the service from the only person in that team who used jQuery for ajax requests 😂